main/main.go

```diff
@@ -155,3 +164,28 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func restoreIBU(client client.Client) error {
+	ibu := &lcav1alpha1.ImageBasedUpgrade{}
+	filePath := filepath.Join(utils.Host, utils.IBUFilePath)
+	if err := lcautils.ReadYamlOrJSONFile(filePath, ibu); err != nil {
+		if os.IsNotExist(err) {
+			return nil
+		}
+		return err
+	}
```

Comment:
```suggestion
	if err := lcautils.ReadYamlOrJSONFile(filePath, ibu); err != nil && !os.IsNotExist(err) {
		return fmt.Errorf("failed to read %s file: %w", filePath, err)
	}
```

Reply:
When the file does not exist, we need to return and skip the rest